src/vs/base/browser/ui/iconLabel/iconlabel.css

```diff
@@ -34,6 +34,10 @@
 	text-overflow: ellipsis;
 }
 
+.sizing-shrink > .monaco-icon-label > .monaco-icon-label-description-container {
```

Comment:
I suggest to move this into tabstitle.css after line 104 where we have a similar rule.